frontend/src/pages/project/Deployments.vue

```diff
@@ -208,8 +211,7 @@ import deviceApi from '@/api/devices'
 import projectApi from '@/api/project'
 import permissionsMixin from '@/mixins/Permissions'
 import ProjectStatusBadge from '@/pages/project/components/ProjectStatusBadge'
-import Alerts from '@/services/alerts'
```

Comment:
This is also imported from `'@/'` so I'd expect it to be after `@/pages` alphabetically by group

Reply:
Yeah, alphabetical wasn't intentional, I generally like grouping by functionality/purpose. Here we (should) have:

- 3rd party libraries
- Vue Stuff
- FF Services
- FF APIs
- FF Components

although, this was a lazy attempt and making that so, and didn't do all of them

Reply:
I think we get that grouping for free, if sorting alphabetically, based on our folder structure, but on this page there's a mix of `@` imports and relative imports.

Reply:
Addressed with https://github.com/flowforge/flowforge/pull/1710/commits/59dc40019580d33b0d373d2fa27bb43cde47b27a